.github/workflows/run-post-submission-jobs.yaml

```diff
@@ -42,32 +81,67 @@ jobs:
         with:
           version: ">=0.0.1"
 
-      - name: Get nowcast date (aka round_id) for target data🕰️
-        # if nowcast_date is not provided, use the most recent Wednesday
-        run: |
-          source ${GITHUB_WORKSPACE}/.github/shared/shared-functions.sh
-          NOWCAST_DATE=$(get_latest_wednesday "${{ inputs.nowcast_date }}")
-          TARGET_NOWCAST_DATE=$(date -d "$NOWCAST_DATE - 91 days" +%Y-%m-%d)
-          echo "NOWCAST_DATE=$NOWCAST_DATE" >> $GITHUB_ENV
-          echo "TARGET_NOWCAST_DATE=$TARGET_NOWCAST_DATE" >> $GITHUB_ENV
-          echo "Creating target data for round $TARGET_NOWCAST_DATE"
-
       - name: Create target data 🎯
         run: |
-          uv run get_target_data.py --nowcast-date=$TARGET_NOWCAST_DATE
+          echo "sequence_as_of: $SEQUENCE_AS_OF"
+          uv run get_target_data.py \
+          --nowcast-date=${{ matrix.nowcast-date }} \
+          --sequence-as-of=$SEQUENCE_AS_OF \
+          --target-data-dir=${{ github.workspace }}
```

Comment:
Now that we're running this job multiple times, write the generated target files to the github workspace working directory and upload them as artifacts (see below).

The final step of the job will download the artifacts from every target run, combine them, and submit a single PR.